docs/modules/reference/pages/daemons/daemon-config-files/event-translator.adoc

```diff
@@ -0,0 +1,42 @@
+[[ref-daemon-event-translate]]
+= Event Translator
+
+The event translator transforms and enhances the data from {page-component-title} events that other daemons generate.
+It enables better integration with external systems, based on data from sources internal and external to {page-component-title}.
+
+For more information, see xref:operation:events/event-translator.adoc[Event Translator].
+
+.Event translator configuration file overview
+[options="header"]
+[cols="2,3,1,1"]
+|===
+| File
+| Description
+| Reload Event
+| Restart Required
+
+| translator-configuration.xml
+| Configure event translator behavior.
+For example, associate an event with a different node.
+| no
+| yes
+
+| Translator.default.events.xml
+| Default events to pass through the translator for enrichment.
+| no
+| yes
+
```

Comment:
I don't have this file in any of my installs.
```suggestion
```

Reply:
I'll remove. I found reference to it in a very old article about the event translator, so added it in, even though it was pretty old in the code. Thanks for confirming.